build.psm1

```diff
@@ -63,6 +63,7 @@ function Start-PSBuild {
         [switch]$ResGen,
         [switch]$TypeGen,
         [switch]$Clean,
+        [switch]$NoPSModuleRestore,
```

Comment:
I would flip the default behavior and make it `-PSModuleRestore`.
Very tiny `Start-PSBuild` should be the bare minimum of things sufficient to run PowerShell.
It also should be incremental-friendly.